setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/CommonFiles.swr

```diff
@@ -12,6 +12,7 @@ folder "InstallDir:MSBuild\Microsoft\VisualStudio\Managed"
   file source="$(VisualStudioXamlRulesDir)Microsoft.Managed.DesignTime.targets"
 
 folder "InstallDir:MSBuild\Microsoft\VisualStudio\Managed"
+  file source="$(VisualStudioXamlRulesDir)AdditionalFiles.xaml"
```

Comment:
Does this also need the localization entries too? Adding them to this file was breaking the tests locally that check for it, but maybe that's a problem on my end?

Reply:
No, the `<XlfInput>false</XlfInput>` entry in the project file means this doesn't get translated. The "Foo.xaml" files are for programatic access to item information, while the "Foo.BrowseObject.xaml" files are for properties exposed to the user; those get translated. There are exceptions, but I wouldn't expect this file to be localized.